djangocms_blog/forms.py

```diff
@@ -116,6 +116,10 @@ def __init__(self, *args, **kwargs):
         ]
         super(PostAdminForm, self).__init__(*args, **kwargs)
         if 'categories' in self.fields:
+            if self.app_config:
+                if self.app_config.url_patterns:
+                    if self.app_config.url_patterns == 'category':
```

Comment:
I would merge all these additional `if` in a single line connecting them with `and`
I don't think we need to check if `url_patterns` exists and we can straight control its value

Reply:
I would also use the setting variable as a check value instead of the string `'category'`